examples/resize/Makefile

```diff
@@ -0,0 +1,66 @@
+
+FLTKCONFIG = ../../fltk-config
```

Comment:
These values aren't going to change during the make, so could be assigned statically with := rather than dynamically with = I guess?